drivers/counter/counter_rpi_pico_pit.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2024 Navimatix GmbH
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include <zephyr/drivers/counter.h>
+#include <zephyr/sys/slist.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct rpi_pico_pit_callback {
+	/** This is meant to be used in the driver and the user should not
```

Comment:
Drop them comment, if its not in an include directory this should always be the case.